test/Elastic.Apm.Elasticsearch.Tests/ElasticsearchTestContainer.cs

```diff
@@ -3,14 +3,18 @@
 // Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information
 
-using DotNet.Testcontainers.Containers.Configurations;
-using DotNet.Testcontainers.Containers.Modules.Abstractions;
+
+using System.Linq;
+using DotNet.Testcontainers.Configurations;
+using DotNet.Testcontainers.Containers;
+using Microsoft.Extensions.Logging;
 
 namespace Elastic.Apm.Elasticsearch.Tests
 {
-	public class ElasticsearchTestContainer : HostedServiceContainer
+	public sealed class ElasticsearchTestContainer : TestcontainerDatabase

```

Comment:
Just out of curiosity: is there a reason to make this class `sealed` now?

Reply:
It's because we set `ContainerPort` in the `.ctor` and `ContainerPort` is `virtual`. So this is to make sure the virtual member call in a constructor is fine.

Reply:
Right, makes sense!